src/Aspire.AppHost.Sdk/SDK/Sdk.in.targets

```diff
@@ -91,52 +87,19 @@
        Text="$(MSBuildProjectName) is a .NET Aspire AppHost project that needs a package reference to Aspire.Hosting.AppHost version 8.2.0 or above to work correctly." />
 
     <!--The following is the list of the RIDs that we currently multitarget for DCP and Dashboard packages-->
     <ItemGroup>
-      <_DashboardAndDCPSupportedTargetingRIDs Include="win-x64" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="win-x86" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="win-arm64" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="linux-x64" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="linux-arm64" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="osx-x64" />
-      <_DashboardAndDCPSupportedTargetingRIDs Include="osx-arm64" />
+      <_DashboardAndDCPSupportedTargetingRIDs Include="win-x64;win-x86;win-arm64;linux-x64;linux-arm64;osx-x64;osx-arm64" />
     </ItemGroup>
 
-    <!-- If running in .NET Core, DOTNET_HOST_PATH is set to point to the dotnet executable being used
-    for the build. -->
-    <PropertyGroup>
-      <_DotNetHostPath>$(DOTNET_HOST_PATH)</_DotNetHostPath>
-    </PropertyGroup>
-
-    <!-- If running on .NET Framework MSBuild, then DOTNET_HOST_PATH won't be set, so we need to construct
-    the path using well-known properties. -->
-    <PropertyGroup Condition="'$(_DotNetHostPath)' == ''">
-      <_DotNetHostDirectory>$(NetCoreRoot)</_DotNetHostDirectory>
-      <_DotNetHostFileName>dotnet</_DotNetHostFileName>
-      <_DotNetHostFileName Condition="'$(OS)' == 'Windows_NT'">dotnet.exe</_DotNetHostFileName>
-
-      <_DotNetHostPath>$(_DotNetHostDirectory)\$(_DotNetHostFileName)</_DotNetHostPath>
-    </PropertyGroup>
-
-    <!-- Call Aspire.RuntimeIdentifier.Tool to get the RIDs for Dashboard and DCP packages -->
-    <Exec Command="&quot;$(_DotNetHostPath)&quot; exec &quot;$(AspireRidToolExecutable)&quot; --runtimeGraphPath &quot;$(BundledRuntimeIdentifierGraphFile)&quot; --supportedRids &quot;@(_DashboardAndDCPSupportedTargetingRIDs -> '%(Identity)', ',')&quot; --netcoreSdkRuntimeIdentifier &quot;$(NETCoreSdkRuntimeIdentifier)&quot;"
-          ConsoleToMSBuild="true"
-          StandardOutputImportance="Low"
-          IgnoreExitCode="true">
-      <Output TaskParameter="ExitCode" PropertyName="_AspireRidToolExitCode" />
-      <Output TaskParameter="ConsoleOutput" ItemName="_AspireRidToolOutput" />
-    </Exec>
-
-    <Error Condition="$(_AspireRidToolExitCode) != 0" Text="Failed to run Aspire.RuntimeIdentifier.Tool. Exit code: $(_AspireRidToolExitCode). Output: @(_AspireRidToolOutput)" />
-
-    <PropertyGroup>
-      <_DashboardAndDcpRID>@(_AspireRidToolOutput)</_DashboardAndDcpRID>
-    </PropertyGroup>
+    <!-- Compute the RID for Dashboard and DCP packages -->
+    <PickBestRid RuntimeGraphPath="$(BundledRuntimeIdentifierGraphFile)" CurrentRid="$(NETCoreSdkRuntimeIdentifier)" SupportedRids="@(_DashboardAndDCPSupportedTargetingRIDs)">
```

Comment:
Could `NETCoreSdkPortableRuntimeIdentifier` be an alternative to doing a runtime graph lookup?

Reply:
maybe? I was mostly focused on translating the behaviors that existed before. I like than an explicit lookup will have a clearer error message than a missing implicitly-defined PackageReference.

Reply:
Most use cases are expressed with portable rids.

Either the portable rid comes from the user, or it's the one from the SDK.

For the latter, rather than mapping NETCoreSdkRuntimeIdentifier via the rid graph, NETCoreSdkPortableRuntimeIdentifier can be used.

Reply:
We can't use `NETCoreSdkPortableRuntimeIdentifier` as there are platforms where that won't be correct. This is how we originally had it, where we'd just use that to infer the package name but it didn't work for some distros where that is more specific and we don't cross-compile for it. As an example of the types of things that would show up if we did this are https://github.com/dotnet/aspire/issues/5486 which this logic was specifically fixing.

Reply:
@joperezr are you sure you were using `NETCoreSdkPortableRuntimeIdentifier`? (Note: this contains: _Portable_.)

I'm looking at https://github.com/dotnet/aspire/pull/5695 and I don't see it.